docs/docs/policies/actions.md

```diff
@@ -0,0 +1,40 @@
+# Actions
+
+Policies support global actions and specific actions per event. The supported actions now are:
+
+- log
+- webhook
+
+## Default Action
+
+The default action is the action executed for every event in the policy:
+
+```
+name: rules events example
+description: This policy shows multiple events tracing
+scope:
+  - global
+defaultAction: log
+rules:
+  - event: dropped_executable
+  - event: security_file_open
+  - event: sched_process_exec
+  - event: close
+```
+
+## Event Action
```

Comment:
after this new title, we should introduce the next YAML
"In addition to a default action, it is also possible to specify a custom action for each event within the policy. If no custom action is specified, an event will trigger the default action."

Not sure if we want to call them 'custom' action tho